test/test_mongo_helpers.h

```diff
@@ -64,25 +64,25 @@ inline warehouse_ros_mongo::MongoQuery& downcastQuery(warehouse_ros::Query::Cons
 template <class T>
 ostream& operator<<(ostream& str, const warehouse_ros::MessageWithMetadata<T>& s)
 {
-  const T& msg = s;
-  str << "Message: " << msg;
+  // const T& msg = s;
```

Comment:
There should really be a way to pretty-print messages. Please add a TODO to fix this

Reply:
A TODO comment is added here.